### PR TITLE
add xmldoc about Seq.rev consume input sequence

### PR DESCRIPTION
**Pull request by enricosada**
ref #902 

phrase is ok?

### CODE REVIEW

File: src/fsharp/FSharp.Core/seq.fsi [hunk 1]
```diff
@@ -957,6 +957,7 @@ namespace Microsoft.FSharp.Collections
         /// <param name="source">The input sequence.</param>
         /// <returns>The reversed sequence.</returns>
         /// <exception cref="System.ArgumentNullException">Thrown when the input sequence is null.</exception>
+        /// <remarks>Consume the whole input sequence before yielding the first element.</remarks>
```
Comment (dsyme) [inline src/fsharp/FSharp.Core/seq.fsi]: Suggest 

    This function consumes the whole input sequence before yielding the first element of the reversed sequence.

**Comment by enricosada** [comment]
used @dsyme phrase

**Comment by enricosada** [comment]
others function who consume the whole input? /cc @vasily-kirichenko @forki @kolektiv

**Comment by forki** [comment]
@7sharp9 identified: sort, foldBack, reduceBack, tryFindBack, findBack, scanBack, tryFindIndexBack, findIndexBack, sortWith, permute, mapFold, mapFoldBack, splitinto

**Comment by enricosada** [comment]
`*Back` it's ok
`*reduce`, `*fold`, `sort`, `scan` ok

but `splitInto` too? why?

**Comment by enricosada** [comment]
added the other functions, thx @forki and @7sharp9 .
@dsyme pharse are ok?

**Comment by dsyme** [comment]
This looks fine.  Since it's just XML documentation I will merge it.

**Pull request merged**